tests/pf_gp_exceptions/enc/enc.c

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/print.h>
+#include <openenclave/internal/tests.h>
+#include "pf_gp_exceptions_t.h"
+
+uint64_t faulting_address;
+uint32_t error_code;
+
+uint64_t test_pfgp_handler(oe_exception_record_t* exception_record)
+{
+    if (exception_record->code == OE_EXCEPTION_PAGE_FAULT)
+    {
+        const int MOV_INSTRUCTION_BYTES = 3;
+        faulting_address = exception_record->faulting_address;
+        error_code = exception_record->error_code;
+        exception_record->context->rip += MOV_INSTRUCTION_BYTES;
+        oe_host_printf(
+            "#PF: addr: 0x%lx, code: %u\n",
+            exception_record->faulting_address,
+            exception_record->error_code);
+    }
+    else if (exception_record->code == OE_EXCEPTION_ACCESS_VIOLATION)
+    {
+        const int LGDT_INSTRUCTION_BYTES = 8;
+        faulting_address = exception_record->faulting_address;
+        error_code = exception_record->error_code;
+        exception_record->context->rip += LGDT_INSTRUCTION_BYTES;
+        oe_host_printf(
+            "#GP: addr: 0x%lx, code: %u\n",
+            exception_record->faulting_address,
+            exception_record->error_code);
+    }
```

Comment:
Line 25 isn't just "else" but rather an else if, so is there a third case?
Suggest adding another else that would (say) fail the test if it activates with any other code.